app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
 
     <string name="winner_title">!! WINNER !!</string>
     <string name="winner_score_title">Your Score </string>
+    <string name="start_new_game">Start a New Game </string>
```

Comment:
The naming convention is:

<activity_related>_<string_title>
in this case:

winner_start_new_game